### DISCUSSION TITLE
Add anchor for label widget

### DISCUSSION THREAD

**Discussion by XxDrexX-cmd** [TomSchimansky/CustomTkinter#210] [Ideas]
Please add **_anchor_** to Label so we can set text to the left of right of the label.

**Comment by avalon60** [comment]
@XxDrexX-cmd  have been going nuts trying to work out what I was doing wrong.  CTkLabel appears to accept anchor but it has no effect. Is this your experience?

**Reply by TomSchimansky** [reply]
It should have an effect when the label is bigger than the actual text. So if you set the label width to 200px and add a small text like "test", it should appear in the middle. You can make the full size of the label visible by setting fg_color="red" for example. And if you set anchor="w", the text should move to the left side.

**Comment by avalon60** [comment]
Hi @TomSchimansky. Thanks for the reply. Here is an example.

If I use tkinter to produce a status bar, this works as expected:

```
import tkinter as tk

app = tk.Tk()
app.geometry('300x200')
app.title("Basic Status Bar")

statusbar = tk.Label(app, text="Loading data…", bd=1, relief=tk.SUNKEN, anchor=tk.W)

statusbar.pack(side=tk.BOTTOM, fill=tk.X)
app.mainloop()
```

![tkinter](https://user-images.githubusercontent.com/89534395/179482908-da8c1c32-1899-4869-ba9b-22c42a406eaf.png)

However when I use customtkinter (modifying the code slightly), The label text is centred:

```
import tkinter as tk
import customtkinter as ctk

app = ctk.CTk()
app.geometry('300x200')
app.title("Basic Status Bar")

statusbar = ctk.CTkLabel(app, text="Loading data…", relief=tk.SUNKEN, anchor=tk.W)

statusbar.pack(side=tk.BOTTOM, fill=tk.X)
app.mainloop()
```
Also the relief=tk.SUNKEN is ignored:

![customtkinter](https://user-images.githubusercontent.com/89534395/179483004-f83d2e4e-1777-4d0b-a0f3-a94cb65b38be.png)


Regards,

Clive

**Reply by TomSchimansky** [reply]
I think you need to upgrade to version 4.5.8, for me it is working:
<img width="412" alt="Bildschirmfoto 2022-07-18 um 12 09 18" src="https://user-images.githubusercontent.com/66446067/179490151-6510cb48-d9bc-4fe1-9795-3a58c36d391a.png">

However, the I would insert 2 whitespaces, to add a some sort of padding on the left, and add a fg_color:
```python
    import tkinter as tk
    import customtkinter as ctk

    app = ctk.CTk()
    app.geometry('300x200')
    app.title("Basic Status Bar")

    statusbar = ctk.CTkLabel(app, text="  Loading data…", relief=tk.SUNKEN, anchor=tk.W,
                             fg_color=("gray82", "gray23"))

    statusbar.pack(side=tk.BOTTOM, fill=tk.X)
    app.mainloop()
```
<img width="412" alt="Bildschirmfoto 2022-07-18 um 12 18 23" src="https://user-images.githubusercontent.com/66446067/179491612-42010a06-0f97-45cb-ac3d-d233a020907c.png">

**Reply by avalon60** [reply]
Blimey @TomSchimansky - that was a prompt response! :o)

Anyway I think you are right. I created that test case on my existing project, which I started a few weeks ago. The version of customtkinter that Pycharm added  was 4.5.1. I just created a new project and tried out the same code and it worked. Before reading your response, I was scratching my head, but following your post I checked the versions. The newer project has installed 4.5.8. 

Thanks for your help.

Regards,

Clive
